app/screens/send-bitcoin-screen/send-bitcoin-details-extra-info.tsx

```diff
@@ -0,0 +1,88 @@
+import { AmountInvalidReason, AmountStatus } from "./payment-details"
+import { useState } from "react"
+import { View } from "react-native"
+import { GaloyWarning } from "@app/components/atomic/galoy-warning"
+import { UpgradeAccountModal } from "@app/components/upgrade-account-modal"
+import { Text } from "@rneui/themed"
+import { AccountLevel } from "@app/graphql/level-context"
+import { useI18nContext } from "@app/i18n/i18n-react"
+import { useDisplayCurrency } from "@app/hooks/use-display-currency"
+import { makeStyles } from "@rneui/base"
+
+export type SendBitcoinDetailsExtraInfoProps = {
+  errorMessage?: string
+  amountStatus: AmountStatus
+  currentLevel: AccountLevel
+}
+
+export const SendBitcoinDetailsExtraInfo = ({
+  errorMessage,
+  amountStatus,
+  currentLevel,
+}: SendBitcoinDetailsExtraInfoProps) => {
+  const [isUpgradeAccountModalVisible, setIsUpgradeAccountModalVisible] = useState(false)
+  const closeModal = () => setIsUpgradeAccountModalVisible(false)
+  const openModal = () => setIsUpgradeAccountModalVisible(true)
+  const { LL } = useI18nContext()
+  const { formatMoneyAmount } = useDisplayCurrency()
+  const styles = useStyles()
+
+  if (errorMessage) {
+    return (
+      <View>
+        <GaloyWarning errorMessage={errorMessage} highlight />
+      </View>
```

Comment:
```suggestion
      <GaloyWarning errorMessage={errorMessage} highlight />
```

may not need a `View`?